src/models/PlatformSchema.ts

```diff
@@ -14,7 +14,13 @@ const PlatformSchema = new Schema({
   credits: {
     type: Number,
   },
-  whitelist: [String],
+  whitelist: {
+    networkName: {
```

Comment:
this is called `assetType` everywhere so please follow that convention so there's no confusion